app/src/test/java/com/androidstudy/mpesa/repo/FakePaymentRepository.kt

```diff
@@ -27,7 +34,11 @@ class FakePaymentRepository : Repository{
         if (shouldReturnNetworkError){
             listener.onNetworkFailure(DarajaException(""))
         }else {
-            listener.onPaymentRequestComplete(PaymentResult("", "", "", "", ""))
+            if (shouldReturnApiError){
+                listener.onPaymentFailure(DarajaException(""))
```

Comment:
Doesn't it make sense to add some values? So it becomes more readable?

DarajaException("Some random exception")

Reply:
Sure, let me fix that